docs/generate-evidence-strings.md

```diff
@@ -79,9 +79,11 @@ ${BSUB_CMDLINE} -K -M 10G \
     --ot-schema    ${BATCH_ROOT}/evidence_strings/opentargets-${OT_SCHEMA_VERSION}.json \
     --out          ${BATCH_ROOT}/evidence_strings/
 
-# Check that the generated evidence strings do not contain any duplicates
-sort ${BATCH_ROOT}/evidence_strings/evidence_strings.json | uniq -c | awk '$1 > 1' > \
-  ${BATCH_ROOT}/evidence_strings/duplicates.json
+# Check that the generated evidence strings do not contain any duplicated evidence strings. For every evidence string, we group the value of fields datatypeId, studyId, targetFromSourceId, variantFunctionalConsequenceId and diseaseFromSourceMappedId, all separated by tabs, sorted and saved at duplicates.json if found duplicated. 
+jq --arg sep $'\t' -jr \
+  '.datatypeId,$sep,.studyId,$sep,.targetFromSourceId,$sep,.variantFunctionalConsequenceId,$sep,.diseaseFromSourceMappedId,"\n"' \
+  ${BATCH_ROOT}/evidence_strings/evidence_strings.json \
+  | sort | uniq -d > ${BATCH_ROOT}/evidence_strings/duplicates.json
```

Comment:
Hopefully it is readable enough: in case a new field is added, we would only need to add a new element (e.g. ``.studyId``) to the ``jq`` string, with a separator (i.e. ``$sep``) on one side to make it look nicer.

Reply:
That's great, thank you @M-casado! Everything's perfectly readable now :)